src/components/ContinuationIn/ManualBusinessInfo.vue

```diff
@@ -237,12 +260,16 @@ import { DatePicker as DatePickerShared } from '@bcrs-shared-components/date-pic
 import { JurisdictionLocation } from '@bcrs-shared-components/enums'
 import { CountriesProvincesMixin } from '@/mixins/'
 import { FormIF } from '@bcrs-shared-components/interfaces'
+import FileUploadPreview from '@/components/common/FileUploadPreview.vue'
```

Comment:
I think you don't need this one any more.

Reply:
Removed